File: Backend/SimpleMeal/SimpleMeal/Models/Supermarket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SimpleMeal.Models
{
    public class Supermarket
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public virtual ICollection<SupermarketProduct> SupermarketProduct { get; set; }
        public bool CanDeliver { get; set; }
        public bool CanPickUp { get; set; }
    }
}

File: iOS/SimpleMeal/SimpleMeal/model/Supermarket.swift
//
//  Supermarket.swift
//  SimpleMeal
//
//  Created by Fhict on 09-01-18.
//  Copyright © 2018 SimpleMeal. All rights reserved.
//

import Foundation
class Supermarket : Decodable{
    let Id : Int
    let Name : String
    let Latitude : Double
    let Longitude : Double
    let CanDeliver : Bool
    let CanPickUp : Bool
    
    init (Id : Int, Name:String, Latitude: Double, Longitude:Double, CanDeliver:Bool, CanPickUp:Bool){
        self.Id=Id
        self.Name=Name
        self.Latitude=Latitude
        self.Longitude=Longitude
        self.CanDeliver=CanDeliver
        self.CanPickUp=CanPickUp
    }
}
